Example/src/AnimatedStyleUpdateExample.tsx

```diff
@@ -29,14 +33,31 @@ function AnimatedStyleUpdateExample(): React.ReactElement {
       }}>
       <Animated.View
         style={[
-          { width: 100, height: 80, backgroundColor: 'black', margin: 30 },
-          style,
+          {
+            width: 100,
+            height: 80,
+            backgroundColor: 'black',
+            margin: 30,
+            marginBottom: 0,
+          },
+          style1,
+        ]}
+      />
+      <Animated.View
+        style={[
+          {
+            height: 80,
+            backgroundColor: 'pink',
+            margin: 30,
+            marginTop: 0,
+          },
+          style2,
         ]}
       />
       <Button
         title="toggle"
         onPress={() => {
-          randomWidth.value = Math.random() * 350;
+          randomWidth.value = Math.random() * 250 + 100;
```

Comment:
I am making changes to example since I've added an extra view updating its with immediately which makes debugging much easier

Reply:
And also to avoid negative values here.